cmake/README.md

```diff
@@ -268,7 +270,7 @@ Variable | Description
 `TBB_IMPORTED_TARGETS`  | All created oneTBB imported targets (not supported for builds from source code)
 
 Starting from [oneTBB 2021.1](https://github.com/oneapi-src/oneTBB/releases/tag/v2021.1), GitHub* release TBBConfig files in the binary packages are located under `<tbb-root>/lib/cmake/TBB`.
-For example, `TBB_DIR` should be set to `<tbb-root>/lib/cmake/TBB`.
+For example, 'TBB_FIND_PACKAGE' should be set to ON and `TBB_DIR` should be set to `<tbb-root>/lib/cmake/TBB`.
```

Comment:
See the comment above.

Reply:
Removed TBB_FIND_PACKAGE from the instructions.